contributing/development/editor/creating_icons.rst

```diff
@@ -38,7 +38,7 @@ Color conversion for light editor themes
 
 If the user has configured their editor to use a light theme, Godot will
 convert the icon's colors based on a
-`set of predefined color mappings <https://github.com/godotengine/godot/blob/4.0.2-stable/editor/editor_themes.cpp#L60-L160>`__.
+`set of predefined color mappings <https://github.com/godotengine/godot/blob/4.4-stable/editor/themes/editor_color_map.cpp#L48-L156>`__.
```

Comment:
```suggestion
`set of predefined color mappings <https://github.com/godotengine/godot/blob/master/editor/themes/editor_color_map.cpp>`__.
```

Given that this is for engine development we really should be linking to the master branch instead of any specific version. This link wouldn't highlight the specific lines but given how small the file is and how easy it is to see what's what I think that works for now.